### DISCUSSION TITLE
Too many requests from your computer

### DISCUSSION THREAD

**Discussion by east-mist** [lirix360/ReadmangaGrabber#30] [Ideas]
Пост 3в1.
1. Наверное, стоит добавить возможность настроить задержку между скачиванием следующей главы.
После скачивания нескольких серий подряд ридманга меня блокирует сообщением  "Too many requests from your computer".
Приложение, к слову, на это не реагирует, просто неестественно быстро заканчивает закачку (а на самом деле просто создает пустые папки).
2. Если в будущем такая возможность появится, то будет здорово добавить что-то вроде очереди загрузки. Сейчас нужно ждать, пока завершится загрузка и лишь потом можно добавить и начать загрузку следующей манги. 
3. Хочу упомянуть еще работу с ошибками. В логе и окне виндовс ошибка то высветиться, только не всегда понятно какая именно глава скачалась не полностью. Например, 
2021-12-17 10:28:19	[ERR]:	(readmanga.go:187):	Ошибка при скачивании страницы: Head "https://h37.rmr.rocks/auto/30/48/76/02.png?t=1639757248&u=0&h=wyWcj1iajfHvVjbhgTnvbA": dial tcp: lookup h37.rmr.rocks: no such host
В таком случае не разобрать, в какой серии и в какой главе случилась ошибка, если только не искать по времени создания папки\файлов.
В остальном приложение просто отличное. Благодарю за работу!

**Comment by lirix360** [comment]
>     1. Наверное, стоит добавить возможность настроить задержку между скачиванием следующей главы.

Это есть в настройках.

**Reply by east-mist** [reply]
Извините, вправду есть.  Не заметил значок настроек -_-)#
Спасибо за помощь!
